src/framework/components/element/component.js

```diff
@@ -142,33 +283,67 @@ pc.extend(pc, function () {
                     child = this._children[i];
                     child.dirtyWorld = true;
                     child._aabbVer++;
-
                 }
+
+                this.element.fire("resize", this.element._width, this.element._height);
+            }
+        },
+
+        _drawDebugBox: function(dt) {
+            var bottomLeft = new pc.Vec3();
```

Comment:
This method is for debugging purposes only – drawLines is very very slow itself anyway, so I don't see a point in avoiding allocs here...